lib/os/__init__.py

```diff
@@ -139,7 +139,7 @@ def close(self, _):
     result = _encode_wait_result(state.Sys())
     if not result:
       return None
-    return state.Pid(), result
+    return state.status 
```

Comment:
well, my Debian machine is not working, and I was testing on a Windows 10 machine (Bash/WSL) :-( and result is always `-1`.